### PR TITLE
Add README.md.

### LINKED ISSUE #1: Readme requested
You should add a readme explaining what this repo is about.

As the composer metadata use the `metapackage` type, this won't impact users as the source will never be downloaded. But it will help people looking at the repo

### PR DESCRIPTION
**Pull request by lloy0076**
Shamelessly "borrowed" `README.md` from:

* https://github.com/php-enqueue/job-queue-flex-pack/blob/master/README.md

Added LICENSE file from:

* https://github.com/api-platform/api-platform/blob/master/LICENSE

It is possible that:

* The year on the LICENCE is wrong (or I've given the code to the WRONG copyright holder)
* The README contents are completely wrong

As for the last point, I am trying to guess what this package does and `api-platform/core` is the most likely primary reason for installing this pack.

Closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Enqueue Job Queue Pack for Symfony Flex
+
+This is a Flex pack to install API Platform, a next-generation web framework designed to easily create API-first projects without compromising extensibility and flexibility.
+
+See the [API Platform's](https://github.com/api-platform/api-platform) github page.
```
Comment (greg0ire) [inline README.md]: I think you should drop the "'s"

File: README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Enqueue Job Queue Pack for Symfony Flex
```
Comment (sroze) [inline README.md]: Might not do the ideal title 😜

**Review by sroze — CHANGES_REQUESTED** [review]

**Comment by greg0ire** [comment]
You can amend your commit message to include `Closes #1` if you want to benefit from autoclose

**Comment by greg0ire** [comment]
It's probably best if you squash your commits

How to do that?

1. `git rebase -i origin/master`, assuming `origin` is a git remote that points to this repository, and not your fork. If you're not sure what your remotes are, run `git remote -vvv`, there should be your fork and the holy/reference/base/origin/whatever-you-call-it repository.
2. A window will show up with many lines, replace `pick` with `fixup` on every line but the first one
3. Close your editor, git should do its magic, and you should end up with one commit
4. Use `git push --force` to overwrite what you already pushed. Don't forget the `--force` option otherwise git will try to merge both things together.

**Review by greg0ire — APPROVED** [review]

**Comment by soyuka** [comment]
we can also squash when merging manually if needed!

**Review by sroze — APPROVED** [review]
If that helps, let's merge then :)

**Pull request merged**